automated install/basic-install.sh

```diff
@@ -160,13 +160,29 @@ if command -v apt-get &> /dev/null; then
     # use iproute
     iproute_pkg="iproute"
   fi
-  # We prefer the php metapackage if it's there
-  if ${PKG_MANAGER} install --dry-run php > /dev/null 2>&1; then
-    phpVer="php"
-  # If not,
-  else
+  # Check for and determine version number (major and minor) of current php install
+  if command -v php &> /dev/null; then
+    phpInsVersion="$(php -v | head -n1 | grep -Po '(?<=PHP )[^ ]+')"
+    echo -e "  ${INFO} Existing PHP installation detected : PHP version $phpInsVersion"
+    phpInsMajor="$(echo "$phpInsVersion" | cut -d\. -f1)"
+    phpInsMinor="$(echo "$phpInsVersion" | cut -d\. -f2)"
+    # Is installed php version supported? (php 5.4 is EOL)
+    if [ "$(echo "$phpInsMajor.$phpInsMinor < 5.5" | bc )" == 0 ]; then
+      phpInsSupported=true
+    fi
+  fi
+  # Check if installed php is unsupported version (5.4 is EOL)
+  if [[ "$phpInsSupported" != true ]]; then
+    # Prefer the php metapackage if it's there
+    if ${PKG_MANAGER} install --dry-run php > /dev/null 2>&1; then
+      phpVer="php"
     # fall back on the php5 packages
-    phpVer="php5"
+    else 
+      phpVer="php5"
+    fi
+  else
+    # Supported php is installed, its common, cgi & sqlite counterparts are deps
+    phpVer="php$phpInsMajor.$phpInsMinor"
   fi
   # We also need the correct version for `php-sqlite` (which differs across distros)
   if ${PKG_MANAGER} install --dry-run ${phpVer}-sqlite3 > /dev/null 2>&1; then
```

Comment:
PHP 5.5 passes the test, but there is no `php5.5-sqlite3` or `php5.5-sqlite` package available in Debian. Only 7.0, 7.1, and 7.2 have packages which include the minor version in the name.